### PR TITLE
Software workshop blog post

### PR DESCRIPTION
**Pull request by karthik**
Please review and recommend changes. Original Google doc is here (has additional comments):  https://docs.google.com/document/d/1tGjdwyFuglsIk6MG18f1jbjDRULfwLERwboiANZRuOs/edit?usp=sharing

### CODE REVIEW

File: content/blog/software-metrics-workshop-report.md [hunk 1]
```diff
@@ -0,0 +1,284 @@
+---
+title: "Report from URSSI workshop on software credit, citation, and metrics"
+date: 2019-03-24T15:17:22-07:00
+author: "Karthik Ram"
+---
+
+
+The mission of the URSSI conceptualization is to understand the challenges that researchers face regarding sustainable research software. One particular challenge identified in our first workshop was the lack of consensus on credit for research software. There is a need to gain a deeper understanding of the concept of credit for research software and how, as a community, we could define and measure impact. To explore this topic is greater detail we conducted a [topic focused workshop](https://github.com/si2-urssi/software-credit#software-metrics-credit-and-citation-workshop) on software metrics and credit at the [National Center for Ecological Analysis and Synthesis (NCEAS)](https://www.nceas.ucsb.edu/) in Santa Barbara on January 23-24, 2019.  We invited [19 experts](https://github.com/si2-urssi/software-credit#confirmed-participants) to participate in a series of discussions and deliver short presentations on their organization's efforts in this space. Below is a summary of our discussions from the two days.
```
Comment (danielskatz) [inline content/blog/software-metrics-workshop-report.md]: AFTER
The mission of the URSSI conceptualization is to understand the challenges that researchers face regarding sustainable research software
ADD: and to design a potential institute that would address these challenges.

Comment (danielskatz) [inline content/blog/software-metrics-workshop-report.md]: "topic is greater" -> "topic in greater"

File: content/blog/software-metrics-workshop-report.md [hunk 2]
```diff
@@ -0,0 +1,284 @@
...
+
+**Discussion topics**
+
+We began the workshop with a group discussion on whether we should focus on software credit and incentives in general or focus specifically on machine actionable/automated ways of assessing software leading which lead to credit measures and incentives.
```
Comment (danielskatz) [inline content/blog/software-metrics-workshop-report.md]: "assessing software leading which lead" -> "assessing software that lead"

File: content/blog/software-metrics-workshop-report.md [hunk 3]
```diff
@@ -0,0 +1,284 @@
...
+
+**Scope of the Software Institute**
+
+What should a software sustainability institute focus on with respect to software credit? Participants advocated for an institute that would support software at all levels, not just software that has been published or archived. Such an institute should support:
+
+
+
+*   New and experienced package authors
+*   Those that rely on scripts as part of their workflow.
+*   How can researchers who share code (that is not packaged in software) get credit for their work in a way that is analogous to credit for methodology?
```
Comment (danielskatz) [inline content/blog/software-metrics-workshop-report.md]: this and the two next items don't really fit the "Such an institute should support:" header - maybe reword?

File: content/blog/software-metrics-workshop-report.md [hunk 4]
```diff
@@ -0,0 +1,284 @@
...
+*   Can researchers get credit for lowering the cost of understanding and extending research (through software)?
+*   The institute could work towards helping scientists get to the next stage from simply sharing scripts. This could take the form of training and support to develop vetted, published software packages that follow community centric best practices. 
+
+    The institute could also help create metrics that would appeal to tenure committees. Software is making its way to[ tenure letters](https://cos.io/blog/are-reproducibility-and-open-science-starting-matter-tenure-and-promotion-review/) and URSSI could play a better role in placing research software in the right context for evaluation. \
```
Comment (danielskatz) [inline content/blog/software-metrics-workshop-report.md]: "would appeal to" -> "would be useful for"
"[ tenure letters]" -> " [tenure letters]"
"play a better role" -> "play a role"

File: content/blog/software-metrics-workshop-report.md [hunk 5]
```diff
@@ -0,0 +1,284 @@
...
+
+
+
+**Lightning talks**
```
Comment (danielskatz) [inline content/blog/software-metrics-workshop-report.md]: The spacing in this section is weird - maybe make all of these elements of a bulleted list?

File: content/blog/software-metrics-workshop-report.md [hunk 6]
```diff
@@ -0,0 +1,284 @@
...
+[FORCE11 Software Citation Implementation WG Update](https://github.com/si2-urssi/software-credit/blob/master/lightning-talks/Daniel%20S.%20Katz%20-%20FORCE11-software-citation-WG-update.pdf) - Daniel S. Katz, Martin Fenner, Neil Chue Hong (Working Group Co-Chairs)
+
+[Implementations and provocations from the asclepias project (2019)](https://github.com/si2-urssi/software-credit/blob/master/lightning-talks/August%20Muench%20asclepias%20URSSI%202019.pdf) - August Muench
+
+[The CodeMeta Project](https://codemeta.github.io/) - Carl Boettiger
+
+[Recommendations on Software and Service Citations from ESIP](https://github.com/si2-urssi/software-credit/blob/master/lightning-talks/jhausman_softservesip_v2.pdf) - Jessica Hausman \
+[CiteAs.org: Discovering and Improving software requests for citation](http://urssi.us/blog/2018/10/01/citeas.org-discovering-and-improving-software-requests-for-citation/) - James Howison
+
+ \
```
Comment (danielskatz) [inline content/blog/software-metrics-workshop-report.md]: remove this \

File: content/blog/software-metrics-workshop-report.md [hunk 7]
```diff
@@ -0,0 +1,284 @@
...
+
+**Discussion topics**
+
+We began the workshop with a group discussion on whether we should focus on software credit and incentives in general or focus specifically on machine actionable/automated ways of assessing software leading which lead to credit measures and incentives. 
+
+We focused the discussion on what an ideal credit scenario might look like and how to better reward software through citation. In particular we tried to unpack the intent of citation in the context of research software:
+
+- Do I want to cite software to **provide credit**
```
Comment (danielskatz) [inline content/blog/software-metrics-workshop-report.md]: maybe add "?" to the end of this and the other 3 items?

File: content/blog/software-metrics-workshop-report.md [hunk 8]
```diff
@@ -0,0 +1,284 @@
...
+
+In the last half-day of the workshop, the group tried to follow the methodology from ["Good Strategy Bad Strategy"](http://goodbadstrategy.com/) by Richard P. Rumelt, to develop a strategy for an institute to follow.  Rumelt says that the kernel of a strategy contains three elements: 
+
+1.	A **diagnosis** that defines or explains the nature of the challenge. A good diagnosis simplifies the often-overwhelming complexity of reality by identifying certain aspects of the situation as critical. 
+
+2.	A **guiding policy** for dealing with the challenge. This is an overall approach chosen to cope with or overcome the obstacles identified in the diagnosis. 
+
+3.	A set of **coherent actions** that are designed to carry out the guiding policy. These are steps that are coordinated with one another to work together in accomplishing the guiding policy.
+
+First, the attendees, working with the overall challenge of making research software more sustainable, determined a set of eight challenges, six of which are related to credit: \
```
Comment (danielskatz) [inline content/blog/software-metrics-workshop-report.md]: remove trailing \

File: content/blog/software-metrics-workshop-report.md [hunk 9]
```diff
@@ -0,0 +1,284 @@
...
+
+**3.	The academic credit model disincentivizes individual contributions to public goods / infrastructure**
+
+**4.	[Disentangle | We conflate] quality and impact of software**
+
+**5.	There is a lack of recognition of the value and importance of software**
+
+**6.	There is a lack of [funding opportunities | stable funding] for maintenance of software that is important but doesn't have a generic market**
+
+**And two of which are not:**
```
Comment (danielskatz) [inline content/blog/software-metrics-workshop-report.md]: remove bolding

File: content/blog/software-metrics-workshop-report.md [hunk 10]
```diff
@@ -0,0 +1,284 @@
...
+
+In addition, it's not clear that open sourcing the software automatically makes it more sustainable.  Some of the group think that the development methodology of open source is important, but that methodology can also be used in more closed environments, such as in a private group or within a company.  This observation led to a statement that closed source but openly developed software is more sustainable than purely open source software.
+
+This discussion led to the issue that URSSI needs to decide how it will deal with commercialization as a path towards sustainability. Will URSSI help/support/aim at commercialization and revenue discovery (while preserving underlying open communities)? In other words, given that the goal of URSSI is sustainable software, what paths towards sustainability should and should not be supported?
+
+Second, the attendees developed a set of guiding policies for an institute, which the groups modified during the breakouts sessions to:
+
+•	Leverage existing organizations for authority, credibility, resources
```
Comment (danielskatz) [inline content/blog/software-metrics-workshop-report.md]: change list to normal bullets/items?

File: content/blog/software-metrics-workshop-report.md [hunk 11]
```diff
@@ -0,0 +1,284 @@
...
+
+•	Coordinate activities rather than start new ones
+
+•	There's no 'one true career path'
+
+Third, the group broke up into three breakout groups to discuss actions around the first six challenges, then formed three different breakout groups to talk about the remaining three challenges. For each challenge, the group determined which guiding principles were relevant, in some cases modifying the principles.  Each group then identified relevant activities, categorized those activities as either short- or long-term, and provided an estimated budget (out of the $1m/year) for those activities (realizing that the other groups would use the remainder of the budget).
+
+While the detailed results follow, the important points that came out of this activity are:
+
+•	In general, actions support more than one challenge. In other words, the sets of actions for each challenge have overlaps with actions from other challenges.
```
Comment (danielskatz) [inline content/blog/software-metrics-workshop-report.md]: change list to normal bullets/items?

File: content/blog/software-metrics-workshop-report.md [hunk 12]
```diff
@@ -0,0 +1,284 @@
...
+
+•	Related to this, while the cost of the set of actions for each challenge if simply added across all the challenges would be about $2.7m-$3m/year which far exceeds the $1m/year budget, the overlaps offer the promise to reduce this cost
```
Comment (danielskatz) [inline content/blog/software-metrics-workshop-report.md]: change list to normal bullets/items?

Comment (danielskatz) [reply content/blog/software-metrics-workshop-report.md]: and do this for all remaining lists

**Comment by danielskatz** [comment]
I've added a bunch of comments, mostly very minor.

I also wonder if we should briefly summarize the overall results in the first paragraph, and then let the rest be a more detailed summary?

**Comment by karthik** [comment]
> I've added a bunch of comments, mostly very minor.

I fixed most of these. 

> I also wonder if we should briefly summarize the overall results in the first paragraph, and then let the rest be a more detailed summary?

I'll work on a short paragraph providing a summary

**Comment by karthik** [comment]
@danielskatz I added a summary. Can you take another quick look at the post?

👉 [Preview of post](http://urssi.us/blog/2019/03/24/report-from-urssi-workshop-on-software-credit-citation-and-metrics/)

**Comment by danielskatz** [comment]
lgtm
